commands/builds/cancel.js

```diff
@@ -0,0 +1,38 @@
+'use strict'
+
+let cli = require('heroku-cli-util')
+let co = require('co')
+let builds = require('../../lib/builds')
+
+async function cancel (context, heroku) {
+  let build = await builds.FindByLatestOrId(heroku, context.app, context.args.build)
+
+  if (build.status !== 'pending') {
+    cli.error(`Can only cancel pending builds. Build ${build.id} has status '${build.status}'`, 1)
+    return
+  }
+
+  let d = heroku.request({
+    method: 'DELETE',
+    path: `/apps/${context.app}/builds/${build.id}`,
+    headers: {'Accept': `application/vnd.heroku+json; version=3.cancel-build`}
+  })
+  await cli.action(`Stopping build ${build.id}`, d)
+}
+
+module.exports = {
+  topic: 'builds',
+  command: 'cancel',
+  needsAuth: true,
+  needsApp: true,
+  description: 'cancel a running build',
+  help: 'Stops executing a running build. Omit BUILD or use "current" in place of BUILD to cancel the latest build.',
```

Comment:
Can we avoid supporting "current"?

Reply:
The support for `current` is in [FindByLatestOrId](https://github.com/heroku/heroku-builds/blob/master/lib/builds.js#L12).
We can remove support for it, but it will also remove it in every other command that uses this method (info and output).

Reply:
If we at least don't document it here, it's less of a breaking change if we later remove it. I don't think we should have two ways of doing the same thing and omitting the argument is the easier option (and, I think, more consistent with other commands).